tests/unit/CourtOrderPoa.spec.ts

```diff
@@ -0,0 +1,35 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import VueRouter from 'vue-router'
+import { Wrapper, createLocalVue, mount } from '@vue/test-utils'
+import { getVuexStore } from '@/store/'
+import CourtOrderPoa from '@/components/common/CourtOrderPoa.vue'
+import { CourtOrderPoa as CourtOrderPoaShared } from '@bcrs-shared-components/court-order-poa/'
+
+Vue.use(Vuetify)
+
+let vuetify = new Vuetify({})
+
+const localVue = createLocalVue()
+const courtOrderNumber = '#court-order-number-input'
+
+localVue.use(VueRouter)
+
+
+describe('CourtOrderPoa component', () => {
+  let store: any = getVuexStore()
+  let wrapper: Wrapper<CourtOrderPoa>
+
+  beforeEach(() => {
+    wrapper = mount(CourtOrderPoa, { store, vuetify })
+  })
+  
+  afterEach(() => {
+    wrapper.destroy()
+  })
+
+  it('renders the components', async () => {
+    expect(wrapper.findComponent(CourtOrderPoa).exists()).toBe(true)
+    expect(wrapper.findComponent(CourtOrderPoaShared).exists()).toBe(true)
+  })
+})
```

Comment:
This test suite is good enough for now.

Please fix the lint errors identified by the PR CI check. (You may be able to run `npm run lint` to fix them automatically, or at least identify them locally.)

Reply:
Linting errors have been fixed. Thanks for all the help!